tests/acceptance/features/emails/super_user/user_create.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Send email at user creation
+  In order to send an email at user creation
+  As a super user
+  I need to create a user
+
+  @super-user @user @add-user @1016-1245 @duplicate
```

Comment:
- [ ] How will this pass since the email is no longer sent when a super-user adds a user?
- [ ] Should not be marked as a duplicate (what is it a duplicate of?)
- [ ] The @1016-1245 here and the next line should be changed to be a unique identifier for the test (current time in MMDD-hhmm format is usually unique, otherwise increase the "minute" value until unique)